### DISCUSSION TITLE
Edward Mãos de Tesourinha de Unha

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#412] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/edward-maos-de-tesourinha-de-unha/

**Comment by angellisy** [comment]
Omodeoso que vontade de pegar e colocar num potinho :hugs:

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/edward-maos-de-tesourinha-de-unha/#comment_1114018440>):

> Omodeoso que vontade de pegar e colocar num potinho 🤗

Colocar num potinho? Se acabar como esse aqui pode dar ruim:



![null](<https://i.postimg.cc/Pfv0hzKr/latest.png>)
